migration/sql-files/028-iterations-parent-path-ltree.sql

```diff
@@ -0,0 +1,22 @@
+CREATE EXTENSION IF NOT EXISTS "ltree";
+
+-- Rename parent_id column
+ALTER TABLE iterations RENAME parent_id to path;
+
+-- Need to convert the path column to text in order to
+-- replace non-locale characters with an underscore
+ALTER TABLE iterations ALTER path TYPE text USING path::text;
+
+-- Need to update values of Iteration's' ParentID in order to migrate it to ltree
+-- Replace every non-C-LOCALE character with an underscore
+UPDATE iterations SET path = regexp_replace(path, '[^a-zA-Z0-9_\.]', '_', 'g');
```

Comment:
Very good point @sbose78 .
As of now there is no UI to create child iterations so I am just hoping that we do not have nested iterations on production. But I will verify once from my local and then add comment here. But yeah thanks for noticing.